### DISCUSSION TITLE
Way to Import generateRoutesFromFiles function

### DISCUSSION THREAD

**Discussion by TheJenos** [nuxt/nuxt#19208] [Questions]
So currently I am working on a module project that needs to generate routes from paths. On the Nuxt 2 I have used createRoutes function from @nuxt/utils but now it's not compatible with Nuxt 3

**Comment by ndragun92** [comment]
You can use

```typescript
const router = useRouter();

router.addRoute({
              name: 'routeName',
              path: `/routePath`,
              component: routeComponent,
              meta: {
                ...yourMeta,
              },
            });
```

**Reply by TheJenos** [reply]
Sorry I need to generate routes from paths

**Reply by ndragun92** [reply]
I dont understand what you mean... Could you give more context or example?

**Reply by TheJenos** [reply]
Currently, I am working on a DND UI builder for Nuxt like Grapesjs. So there, I wanna show all the pages that I have currently on the project. Last time I used createRoutes function from @nuxt/utils to generate routes for Nuxt 2. But now I notice on the Nuxt 3 they had to change the folder name structure. So went through the nuxt/nuxt codebase and I found a [generateRoutesFromFiles  function](https://github.com/nuxt/nuxt/blob/fea495fb6c06056bdac13af96cf22472ae415bb3/packages/nuxt/src/pages/utils.ts#L50). But the issue is I can't import this from my module.
